website/docs/matchers/Array.mdx

```diff
@@ -105,3 +105,14 @@ Use `.toSatisfyAny` when you want to use a custom matcher by supplying a predica
   expect([2, 4, 8, 12]).not.toSatisfyAny(isOdd);
 });`}
 </TestFile>
+
+### .toBeInRange(min, max)
+
+Use `.toBeInRange` when you want to check if the given array contains numbers within the specified range.
```

Comment:
I'd include that min is exclusive, and max is inclusive here.

Reply:
Not sure if @mattphillips has been notified because this comment is edited.

Reply:
An example of how to word the inclusive/exclusive behavior is the toBeWithin documentation
> Use `.toBeWithin` when checking if a number is in between the given bounds of: start (inclusive) and end (exclusive).

Reply:
@keeganwitt I have updated the description now.

Reply:
@keeganwitt I'm not able to request a review from @mattphillips. Could you please help here?

Reply:
I was going to wait another day-ish, and if he doesn't respond, I'll just merge it.

Reply:
@keeganwitt Could you please merge this PR to main branch if all looks good to you?

Reply:
Yea. We can always rename before cutting the next release if there's a better name that we should have used.